.github/workflows/php.yml

```diff
@@ -87,11 +87,11 @@ jobs:
       matrix:
         # https://endoflife.date/php
         php:
-          - 8.1 # EOL: November 25th, 2024
-          - 8.2 # EOL: December 8th, 2025
+          - 8.2 # EOL: December 31th, 2026
```

Comment:
@AndriiAndreiev it looks like you're using `Security Support`, which should include `8.1` until end of 2025. We might want to continue supporting that in our GH test matrix.